src/core/authentication/ory.strategy.ts

```diff
@@ -50,6 +50,7 @@ export class OryStrategy extends PassportStrategy(Strategy, 'oathkeeper-jwt') {
     if (checkIfTokenHasExpired(payload.exp)) {
       throw new TokenException(
         'Access token has expired!',
+        LogContext.AUTH,
```

Comment:
I don't believe you should be passing `LogContext.AUTH` for `TokenException` it's always with that context. It should be provided in the exception itself.